flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/batch/ExplainTest.scala

```diff
@@ -119,4 +120,25 @@ class ExplainTest
       "../../src/test/scala/resources/testUnion1.out").mkString.replaceAll("\\r\\n", "\n")
     assertEquals(result, source)
   }
+
+  @Test
+  def testProjectMerge(): Unit = {
```

Comment:
I don't think we need to add a test to `ExplainTest.scala`. The other plan tests are sufficient to validate that the desired plans are generated.